chainer/distribution.py

```diff
@@ -0,0 +1,260 @@
+import copy
+
+
+class Distribution(object):
+
+    """Interface of Distribution
+
+    `Distribution` is a bass class to treat probability distributions.
+    When initialization, it takes parameter as input.
```

Comment:
Could you also emphasize that the outputs of every method and property that computes them from `chainer.Variable` can basically be differentiated?